### PR TITLE
[WIP] Scripts/Ulduar: Thorim implementation

### PR DESCRIPTION
**Pull request by ariel-**
Deprecated, please go to #15008

### CODE REVIEW

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp [hunk 1]
```diff
@@ -22,22 +22,221 @@
...
+    EVENT_SUMMON_ADDS,
+    EVENT_BERSERK,
+    EVENT_UNBALANCING_STRIKE,
+    EVENT_CHAIN_LIGHTNING,
+    EVENT_TRANSFER_ENERGY,
+    EVENT_RELEASE_ENERGY
+};
+
+#define EMOTE_BARRIER                           "Runic Colossus surrounds itself with a crackling Runic Barrier!"
+#define EMOTE_MIGHT                             "Ancient Rune Giant fortifies nearby allies with runic might!"
```
Comment (Shauren) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp]: wat

Comment (ariel-) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp]: Is this ok?

```c++
enum Broadcasts
{
    EMOTE_RUNIC_BARRIER                         = 33267,
    EMOTE_RUNIC_FORTIFICATION                   = 33523
};
// [...]
me->TextEmote(EMOTE_RUNIC_BARRIER, nullptr, true);
```

Comment (Nyeriah) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_thorim.cpp]: No.

```cpp
Talk(EMOTE_RUNIC_BARRIER);
```

**Comment by Aokromes** [comment]
@ariel- jump into irc :P

**Comment by joschiwald** [comment]
I won't accept this pr, some time ago i used the princecreed script as base for an rewrite, my script is much better than this, almost each line changed, most things implemented (one example, the lightning field) sniffs used in many places, some things still guessed values, when i get home i'll share it to you, so you can test it (just need to extract from my big ulduar rewrite)

**Comment by ariel-** [comment]
@joschiwald Wait for it...

**Comment by joschiwald** [comment]
come into irc

**Comment by joschiwald** [comment]
new implementation on https://github.com/joschiwald/TrinityCore/commits/thorim

**Pull request closed without merging**